R/pkg/inst/extdata/spark_download.csv

```diff
@@ -0,0 +1,2 @@
+"url","default"
+"http://apache.osuosl.org",TRUE
```

Comment:
Yeah, using apache dynamic mirror is great as long as the address does not change. The reason I was thinking of maintaining a (remote) csv file is that we are able to apply any change of the mirror sites without touching the R package...